owncloudApp/src/main/java/com/owncloud/android/MainApp.java

```diff
@@ -264,18 +233,25 @@ public static String getUserAgent() {
         return String.format(appString, version);
     }
 
-    public static boolean isBeta() {
-        boolean isBeta = false;
-        try {
-            String packageName = getAppContext().getPackageName();
-            PackageInfo packageInfo = getAppContext().getPackageManager().getPackageInfo(packageName, 0);
-            String versionName = packageInfo.versionName;
-            if (versionName.contains(BETA_VERSION)) {
-                isBeta = true;
-            }
-        } catch (PackageManager.NameNotFoundException e) {
-            e.printStackTrace();
+    public static boolean isDeveloper() {
+        return mDeveloper;
+    }
+
+    public void readIsDeveloper() {
+        mDeveloper = BuildConfig.DEBUG || PreferenceManager.getDefaultSharedPreferences(getApplicationContext())
+                .getInt(CLICK_DEV_MENU, 0) > CLICKS_NEEDED_TO_BE_DEVELOPER;
+
+
+        if (isDeveloper()) {
+
+            String dataFolder = getDataFolder();
+
+            // Set folder for store logs
+            Log_OC.setLogDataFolder(dataFolder);
+
+            Log_OC.startLogging(Environment.getExternalStorageDirectory().getAbsolutePath());
+            Log_OC.d(BuildConfig.BUILD_TYPE, "start logging " + BuildConfig.VERSION_NAME + " " + BuildConfig.COMMIT_SHA1);
```

Comment:
Line too long

Reply:
This is the reason, why I vote for 150: there is enough place !
![image](https://user-images.githubusercontent.com/3314607/54336266-df92f900-462b-11e9-8c5a-43fe36b4c88b.png)

But you reduce to 120

This seems to be better ? Tastes are different
![image](https://user-images.githubusercontent.com/3314607/54336331-0c471080-462c-11e9-94dd-c1718cb6da28.png)

Reply:
We'd rather 120 because of how it looks in GH as well, the next screenshot is with more than 120 columns:

![Screenshot_2019-03-14_at_09_38_49](https://user-images.githubusercontent.com/15261363/54342588-276e4c00-463d-11e9-9326-fa96fe8d8654.png)